src/routes/api/articles/articleController.js

```diff
@@ -49,4 +49,36 @@ export default class Article {
       });
     }
   }
+
+  /**
+ * @static
+ * @param {obj} request
+ * @param {ogj} response
+ *  @param {next} next
+ * @returns {void}
+ * @memberof Article class
+ */
+  static async getAllArticles(request, response, next) {
+    try {
+      let offset = 0;
+      const limit = 20;
```

Comment:
Remove this static declaration and pass in offset and limit as query parameters. Your implementation should return all articles if the parameters are not specified.